packages/frontend/web-ui/src/common/hooks/useDisplayDialog.ts

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import { DisplayDialog } from '../models/DisplayDialog';
+
+export const useDisplayDialog = (): DisplayDialog => {
```

Comment:
Isn't this hook an overkill? You can do the same with a `useState<boolean>`

Reply:
I take as an example the useShowPassword hook. It isn't the same, but it's possibly that you have the reason. Then, in your opinion, is better to put this in the same component?